server/jetstream_api.go

```diff
@@ -307,6 +307,9 @@ const (
 	// JSAdvisoryServerRemoved notification that a server has been removed from the system.
 	JSAdvisoryServerRemoved = "$JS.EVENT.ADVISORY.SERVER.REMOVED"
 
+	// JSAdvisoryAPIRateLimit notification that a server has reached the JS API rate limit.
+	JSAdvisoryAPIRateLimit = "$JS.EVENT.ADVISORY.API.RATE_LIMIT"
```

Comment:
I wonder if this should fall under API AUDIT that already exists?

Reply:
not really an audit trail but otoh might be worth knowing when things are dropped in auditing, but introducing a new type of message on that subject my trip up some things people have written

Reply:
I was suggesting use same format, nil request and response that essentially says overloaded, etc..

Reply:
Ah no I would rather have a specific advisory on specific subject. The api audit tends to spam a lot especially when busy and then you miss these. 

On a separate subject and format you will be able to just listen on those for example.